package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Stethoscope",
```

Comment:
It looks like there is a lot of churn with styling. This might be a good time to incorporate `husky` and `lint-staged` to format changed files on commit :)

Something like this, but using `standard` instead of `prettier-standard`, and doing `eslint --fix`
https://code.likeagirl.io/keep-your-codebase-neat-and-tidy-with-prettier-standard-lint-staged-and-husky-9f9a6b1d4f72

Reply:
I was interested in testing out Github actions, so I setup https://github.com/Netflix-Skunkworks/stethoscope-app/blob/master/.github/workflows/validate-push.yml

Reply:
oh, interesting! I wonder if this can modify the code to automatically run the lint and standard fixes, or if it just opens issues on the PR